media_management_api/urls.py

```diff
@@ -1,12 +1,15 @@
 from django.urls import include, path
 from django.contrib import admin
 from django.views.generic.base import RedirectView
+from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView
 
 # Wire up our API using automatic URL routing.
 # Additionally, we include login URLs for the browsable API.
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/auth/', include('media_management_api.media_auth.urls')),
+    path('api/token/', TokenObtainPairView.as_view(), name='token_obtian_pair'),
```

Comment:
Not to nitpick but I think `obtain` is misspelled.

I also have a question about where the tokens are stored, I didn't see them get added to the database. Is it safe to assume it will be stored in local storage on the client?

Otherwise every works great!

Reply:
Oy 🤦 thanks for the catch.

Great question. JWTs actually don't need to be stored in a DB, since the tech uses a hashing algorithm to encode and decode the tokens on the backend. That's actually one of the main benefits of JWTs. Any client leveraging this auth method should be storing the access and refresh tokens in some way (often local storage). Here's a blog that I found useful in understanding/setting it up:
https://simpleisbetterthancomplex.com/tutorial/2018/12/19/how-to-use-jwt-authentication-with-django-rest-framework.html